src/controllers/legacycontrollersettings.h

```diff
@@ -500,8 +661,33 @@ class LegacyControllerFileSetting
         return QJSValue(stringify());
     }
 
+#ifdef MIXXX_USE_QML
+    void setValue(const QJSValue& rawValue) override {
+        QUrl path = rawValue.toVariant().value<QUrl>();
+        QFileInfo value;
+        if (!path.isValid()) {
+            value = QFileInfo();
+        } else {
+            value = QFileInfo(path.toLocalFile());
+            DEBUG_ASSERT(value.exists());
```

Comment:
We should only assert things that can go wrong. A file can be missing. Please introduce a proper error handling if that is required at this low level.

Reply:
For my understanding it is also created form the xml file. So lets make it a warming so that we also see the issue in a release build, and it is not fatal in debug builds.